### ISSUE TITLE
Are cells supposed to be editable?

### ISSUE THREAD

**Issue by andr-c** [dbeaver/cloudbeaver#1501]
I was unable to edit cells as documented here - https://github.com/dbeaver/cloudbeaver/wiki/CloudBeaver-overview#view-and-edit-table-data. Nothing happens when I double click a cell, nor menu option to edit is shown up.
I was trying stock docker image, as well as demo.cloudbeaver.io. 
Do I need to do some extra steps for configuration to make editing possible?

**Comment by kseniaguzeeva** [comment]
Thank you for your interest in CloudBeaver. 
Could you please give me more details to identify the problem:
1. What database do you use?
2. Does your table have a primary key?
3. Do you open a table via the Data Editor or the SQL Editor? If it is the SQL Editor, could you give me a query example?
4. Could you also tell me the CloudBeaver version?

Thank you in advance.

**Comment by andr-c** [comment]
Thanks very much for your reply! and sorry for silly question - I have restarted the docker container and re-created the connection, and somehow today it worked for me.
It's not clear to me what have been changed.

I'm using:
1. PostgreSQL 13.2 (Debian 13.2-1.pgdg100+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 8.3.0-6) 8.3.0, 64-bit

2. Yes

3. Yes, I was using the data view which is opened after double click on a table.

4. Product name
CloudBeaver CE Server
Product description
Cloudbeaver Web UI Application
Build time
February 20, 2023
Backend version
22.3.5.202302200914
Frontend version
22.3.5.202302200918

Run with the command:
docker run --name cloudbeaver --rm -p 8080:8978  dbeaver/cloudbeaver

I observed the same read-only behavior at https://demo.cloudbeaver.io/#/   Posgres SQL / postgress / test / news table 
hence was wondering if editing is implemented yet
